analysis/README.md

```diff
@@ -0,0 +1,38 @@
+# Distbench analysis script
+
+The distbench analysis tool is a python script developed to transform the results from the native distbench proto format to different formats that can be used for compabality with existing analysis tools.
+
+## How to use it
+
+To use this script, numpy should be installed, one can do this on the linux terminal with the following command:
+
+```bash
+sudo apt install python3-numpy
+```
+
+The script generates a directory tree with summaries across all the clients and services.
+
+to invoke the script use the follwing command line:
+
+```bash
+    ./results_conversion -- --output_directory=output_directory --input_file=name_of_the_file
```

Comment:
this command line does not work. the "--" is leftover from a prior command that was running under bazel.